### DISCUSSION TITLE
中文文献不能跳转，英文可以，会是什么问题造成的？

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#229] [Q&A]
您好，

因为`XelaTeX`爆内存，因此采用`LuaLaTeX-Biber-LuaLaTeX-LuaLaTeX`编译。编译后看起来一切正常，但是点击中文文献作者名和年无法跳转，什么都不动，点击英文可以跳转到文末参考文献。会是什么原因呢？

谢谢。

**Comment by earthmoon** [comment]
`xelatex`没有这问题，`lualatex`造成的，算了，还是用`xelatex`吧。

**Comment by SwitWu** [comment]
什么爆内存？

**Reply by earthmoon** [reply]
文档大了，提示内存不够，网上各种设置`xelatex`内存的，有时够有时又不够。也不知道那些设置的数值是怎么个计算法。

**Reply by SwitWu** [reply]
多大的文档啊

**Reply by earthmoon** [reply]
300页，`tabularray`貌似特别占内存，制表慢。

**Reply by SwitWu** [reply]
`tabularray` 是这样的，非必要的情况下还是用传统的表格吧

**Reply by earthmoon** [reply]
但是`tabularray`好用，哪知道有这么大缺点。
